numerics/gradient_descent_body.hpp

```diff
@@ -103,7 +103,7 @@ double Zoom(double α_lo,
             bool& satisfies_strong_wolfe_condition) {
   std::optional<Scalar> previous_ϕ_αⱼ;
   satisfies_strong_wolfe_condition = true;
-  VLOG(1) << "Zoom over: [" << α_lo << " (" << ϕ_α_lo << "), "
+  LOG(INFO) << "Zoom over: [" << α_lo << " (" << ϕ_α_lo << "), "
```

Comment:
Do we want these logs permanently?

Reply:
Let’s keep them for now, they come from the press of a button, so they are probably less noisy than the pile up stuff we log whenever a Kerbal bumps into something, or than the logs about the prediction timing out.